src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -208,6 +148,143 @@ public void basicTest() throws Exception {
 
     @Test
     public void datasetTest() throws Exception {
+        Dataset defaultDataset = readDataset("ingest-test-dataset.json");
+        Dataset eastDataset = readDataset("ingest-test-dataset-east.json");
+        List<Object[]> testCases = Arrays.asList(
+            new Object[]{defaultDataset, testConfig.getIngestbucket(),
+                "BiqQuery datasets are instantiated in us-central1 by default."},
+            new Object[]{eastDataset, testConfig.getNonDefaultRegionIngestBucket(),
+                "BiqQuery datasets can be set to the non-default region."});
+
+        for (Object[] tuple : testCases) {
+            String targetPath = "scratch/file" + UUID.randomUUID().toString() + "/";
+            Dataset dataset = (Dataset) tuple[0];
+            String bucket = (String) tuple[1];
+            String regionMessage = (String) tuple[2];
+
+            String region = dataset.getDatasetSummary().getStorage().stream()
```

Comment:
It could be worth turning this into a function since we need to do this in several places